plugins/todo/README.md

```diff
@@ -0,0 +1,9 @@
+# @backstage/plugin-todo
+
+This plugins lists `// TODO` comments in source code. It currently exports a single component extension for use on entity pages.
+
```

Comment:
You use a specific parenthetical method to elevate the author of a TODO.  I'll be honest and say I've not seen or experienced projects using that convention before, and Backstage was the first I'd seen it.  It seems this plugin will help elevate that, which is awesome.

Given that, I might suggest this README needs a wee bit of instructions to the expected format it will use?  It could just be me, and interested to hear from others, but it's just something I haven't seen before.

I use a [VS Code Extension called TODO Tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree), and I checked and even it doesn't have a 1st level concept of this, even if it will allow parenthesis to be elevated into the TODO comment.

As an additional point, looking at one of my squad's repos in TODO tree I see this:

![image](https://user-images.githubusercontent.com/33203301/110725073-ecc82b80-81e4-11eb-8b51-b1f0d05459d8.png)

So you can see just as convention, we happened to also log a potential bug in the code, and TODO Tree identified that and even elevated it as a BUG.  I wonder if this plugin could elevate more than just TODO - even if as a first level the word "TODO" is elevated to a potential string list, and in some future refactor, it could be configured for teams to tweak it. (in fact you'll see TODO Tree lets you identify other tags, but the key defaults it uses are TODO, FIXME, and BUG. (not suggesting we try to align this plugin to only that plugin, just using it as another example of code that elevates these!)

Reply:
Yep, planning to show the tags as well and figure out formats a bit more. Parsing is based on https://github.com/pgilad/leasot and I don't know exactly what that supports just yet :grin: